spec/features/admin/accounting/transaction_flow_spec.rb

```diff
@@ -250,6 +251,32 @@
       click_on txn.txn_date.strftime("%B %-d, %Y")
       expect(page).to have_content("icecream")
     end
+
+    describe "an admin can edit a managed txn" do
+      let!(:txn) do
+        create(:accounting_transaction,
+               project_id: loan.id, description: "I love icecream", division: division, managed: true)
+      end
+
+      scenario "edit managed transaction as admin" do
+        visit admin_loan_tab_path(loan, tab: "transactions")
+        click_on txn.txn_date.strftime("%B %-d, %Y")
+        expect(page).to have_content("Edit")
+      end
+    end
+
+    describe "an admin cannot edit a non-managed txn" do
+      let!(:txn) do
+        create(:accounting_transaction,
+               project_id: loan.id, description: "I love icecream", division: division, managed: false, loan_transaction_type_value: :disbursement)
```

Comment:
Please fix. I would do it like this:

```
create(:accounting_transaction, project_id: loan.id, description: "I love icecream", division: division, 
                                managed: false, loan_transaction_type_value: :disbursement)
```

since I prefer not having super tall & narrow blocks. But whatever passes the linter is fine with me.